tests/testthat/test_data/job_list_zero_items.txt

```diff
@@ -0,0 +1 @@
+NULL
```

Comment:
I did not realize that this file merely output a NULL, but I propose we leave it as it allows all of our tests to follow the same pattern.